mozilla/foundation.mozilla.org
# Make header and name the same in factory code

Closes #6569 

**Link to sample test page**: https://foundation-s-6569-petit-vznhd1.herokuapp.com/cms/snippets/wagtailpages/petition/

Steps to test:
* Click the link above. 
* Click into a few petitions 
* The name and header should be the same values
